packages/components/app/styles/components/dropdown.scss

```diff
@@ -413,8 +421,88 @@ $hds-dropdown-toggle-border-radius: 5px;
   }
 }
 
+// HDS::DROPDOWN::LIST-ITEM::TITLE
+
 .hds-dropdown-list-item--title {
   padding: 10px 16px 4px;
   color: var(--token-color-foreground-strong);
 }
 
+// HDS::DROPDOWN::LIST-ITEM::CHECKMARK
+
+.hds-dropdown-list-item--checkmark {
+  a.hds-dropdown-list-item__interactive,
+  button.hds-dropdown-list-item__interactive {
+    // disable background on focus and active states
+    &:focus,
+    &:active,
+    &.mock-focus,
+    &.mock-active {
+      color: var(--token-color-foreground-action-active);
+
+      &::after {
+        background-color: inherit;
+      }
+    }
+  }
+
+  .hds-dropdown-list-item__interactive-text {
+    flex: 1;
+  }
+}
+
+.hds-dropdown-list-item--checkmark-selected {
+  a.hds-dropdown-list-item__interactive,
+  button.hds-dropdown-list-item__interactive {
+    padding-right: 9px; // reset to default
```

Comment:
I _think_ you don't need this anymore, can you double check?
```suggestion
```

Reply:
Indeed, this was leftover – removed it